GW2EIEvtcParser/ParserHelpers/IDs/DamageModifierIDs.cs

```diff
@@ -358,5 +358,34 @@ public static class DamageModifierIDs
     public const int Mod_Brutality = 345;
     public const int Mod_PowerBlock = 346;
     public const int Mod_PhantomPain = 347;
+    public const int Mod_ExhilaratingEphemera = 348;
+    public const int Mod_RitualistsShroud = 349;
+    public const int Mod_ElementalBalanceOutgoing5 = 350;
```

Comment:
OutgoingX/IncomingX patterns should be merged:
ex 
Mod_ElementalBalanceOutgoing5_Incoming10
Mod_ElementalBalanceOutgoing10_Incoming5
as they are the same modifier